packages/math-input/src/components/iconography/index.js

```diff
@@ -2,46 +2,44 @@
  * A directory of autogenerated icon components.
```

Comment:
Was the file that generates this file updated?

Reply:
No. I'm going to work on a PR today to update the generation script.